lib/filtrex/condition.ex

```diff
@@ -81,6 +90,19 @@ defmodule Filtrex.Condition do
     if result, do: result, else: {:error, "Unknown filter key '#{key_with_comparator}'"}
   end
 
+  @doc "Dumps condition into map"
+  @spec dump(any_condition) :: Map.t
+  def dump(condition) do
+    condition
+    |> put_dump_value
+    |> Map.from_struct
+    |> Enum.reduce(%{}, fn ({key, value}, acc) ->
+      Map.put(acc, Atom.to_string(key), value)
+    end)
+    |> Map.update!("type", &(Atom.to_string(&1)))
```

Comment:
```suggestion
    |> Map.update!("type", &Atom.to_string/1))
```

This shorter syntax should work the same. See other places where this could also be used.